packages/core/upload/server/services/extensions/content-manager/entity-manager.js

```diff
@@ -64,7 +64,10 @@ const addSignedFileUrlsToAdmin = async () => {
        * @param {string} uid
        * @returns
        */
-      const mapEntity = (entity, uid) => signEntityMedia(entity, uid);
+      const mapEntity = async (entity, uid) => {
+        const signedEntity = await signEntityMedia(entity, uid);
+        return entityManager.mapEntity(signedEntity, uid);
```

Comment:
Am I right in thinking this wont change the response? But that we should call the parent function as that is a pattern that all decorators should follow?

Reply:
yes exactly! it will not change the response but call the parent function or any other decorators that it might have.